server/main.go

```diff
@@ -112,11 +112,24 @@ func getState(c web.C, w http.ResponseWriter, r *http.Request) {
 	encoder.Encode(states)
 }
 
+// Returns info about the target.
+//
+// If a target is particularly large and will therefore take a long time
+// to download, we want to allow clients to supply their own copy of it.
+// In this case this endpoint will return the command that they should
+// run in order to fuzz it.
+func getTargetMeta(c web.C, w http.ResponseWriter, r *http.Request) {
```

Comment:
This new route is kind of the key to the whole thing, I imagine we might want to add extra stuff here in the future.